ci/push_docker.sh

```diff
@@ -23,14 +22,27 @@ function release() {
             return 1
         fi
         tag_exact="${DOCKER_REPO}:${weaviate_version}"
+  else
+    pr_title="$(echo -n "$PR_TITLE" | tr '[:upper:]' '[:lower:]' | tr -c -s '[:alnum:]' '-')"
+    tag_preview="${DOCKER_REPO}:preview-${pr_title}-${git_hash}"
   fi
 
-  args=("--build-arg=GITHASH=$git_hash" "--platform=linux/amd64,linux/arm64" "--target=weaviate" "-t=$tag_latest" "--push")
+  # args=("--build-arg=GITHASH=$git_hash" "--platform=linux/amd64,linux/arm64" "--target=weaviate" "-t=$tag_latest" "--push")
+  args=("--build-arg=GITHASH=$git_hash" "--platform=linux/amd64" "--target=weaviate" "-t=$tag_latest" "--push")
   if [ -n "$tag_exact" ]; then
+    # exact tag on master
     args+=("-t=$tag_exact")
   fi
+  if [ -n "$tag_preview" ]; then
+    # preview tag on PR builds
```

Comment:
also on push? Eg what would happen after a merge to master?

Reply:
Right now, no report yet. Would like to fix that in a separate PR.